README.md

```diff
@@ -460,7 +460,11 @@ Other Style Guides
     ];
 
     // good
-    const arr = [[0, 1], [2, 3], [4, 5]];
+    const arr = [
```

Comment:
this is intentional; this array *doesn't* have multiple lines, so there's no need to use line breaks - that's why it's a "good" example.